app/gen-server/lib/DocApiForwarder.ts

```diff
@@ -104,7 +104,7 @@ export class DocApiForwarder {
     url.pathname = removeTrailingSlash(docWorkerUrl.pathname) + url.pathname;
 
     const headers: {[key: string]: string} = {
-      ...getTransitiveHeaders(req),
+      ...getTransitiveHeaders(req, { includeOrigin: false }),
```

Comment:
I wonder if I should not actually include the `Origin` header here.

Reply:
I tried switching `includeOrigin` to `true` here, but the form submission does not work anymore. Probably worth to understand why.